### PR TITLE
wordsmithing

### PR DESCRIPTION
**Pull request by hunterowens**

### CODE REVIEW

File: dags/public-health/covid19/get-help-to-esri.py [hunk 1]
```diff
@@ -494,7 +494,7 @@ def format_table(row):
     # Create the email body.
     entry = f"""<b>{shelter_name}</b><br>
     <i>Council District {district}</i><br>
-    <i>Latest update: {last_update}</i><br><br>
+    <i>Last Change in Program: {last_update}</i><br><br>
```
Comment (ian-r-rose) [inline dags/public-health/covid19/get-help-to-esri.py:497]: "Program" seems a bit technical to me. We know that GetHelp refers to them as programs, but does everyone?

Comment (hunterowens) [reply dags/public-health/covid19/get-help-to-esri.py:497]: yeah. Last change at site?

Comment (ian-r-rose) [reply dags/public-health/covid19/get-help-to-esri.py:497]: Actually, I'm (just now) hearing from GetHelp that they were reporting `lastUpdated` as the last time the program was changed, not the time of last bed activity, and that they will be changing it to the latter this afternoon.

So perhaps we don't need to do anything here? Or maybe we should still clarify the language regardless.

Comment (ian-r-rose) [reply dags/public-health/covid19/get-help-to-esri.py:497]: I like "Last change at site". 👍 from me

**Pull request merged**